app/routes/groups_.$groupId.tsx

```diff
@@ -3,20 +3,132 @@ import { useParams, useLoaderData, Link } from '@remix-run/react';
 import { db } from '~/modules/database/db.server';
 import { json } from '@remix-run/node';
 
+import { Button } from '~/components/ui/button';
+
+import { Image } from '~/components/ui/images';
+
 export const loader: LoaderFunction = async ({ params }) => {
   const group = await db.group.findFirstOrThrow({ where: { id: params.groupId } });
   return json({ group });
 };
 
 export default function GroupRoute() {
   const data = useLoaderData<typeof loader>();
-
   return (
-    <>
-      <Link to="/groups">back to groups</Link>
-      <h1>{data.group.name}</h1>
-      <p>{data.group.description}</p>
-    </>
+    <div className="bg-primary text-white flex flex-col items-center">
+      <div className="md:flex md:flex-col md:max-w-screen-xl">
+        <div className="p-2 justify-center md:p-10 md:flex md:flex-row md:border-b-2 md:border-b-white">
+          <div className="m-auto rounded-full h-[300px] w-[280px] justify-center items-center md:m-8">
+            {!data.group.imgUrl && !data.group.ImgAlt ? (
```

Comment:
I am surprised that TypeScript does not complain here that these properties do not exist! 😅 

Happy to keep it like that but also feel free to add them already to the prisma schema file!

Nit: 

```suggestion
            {!data.group.imgUrl && !data.group.imgAlt ? (
```